src/Components/Publishing/Sections/ImageSetPreview/ImageSetLabel.tsx

```diff
@@ -14,20 +14,26 @@ export const ImageSetLabel = (props: ImageSetPreviewProps) => {
   const label = images.length === 1 ? "Image" : "Images"
   const imageCount = `${images.length} ${label}`
   const primaryTitle = title ? title : imageCount
+  const textColor = color !== "white" ? color : undefined
```

Comment:
See previous comment

Reply:
this is confirming the value of the input, not declaring it -- i believe `color("white100")` resolves to `"white"` so this should work as expected.